docs/documentation/custom_modules/add_submodels.rst

```diff
@@ -110,18 +110,20 @@ To allow a simpler replacement of your submodels, you will need to use the
 This has the same behavior as the previous one, but the second one will allow substitution of
 submodels, as shown in next part.
 
-In details, :code:`CountWingAtoms` is declared as a submodel that fulfills the role of "wing atom
+In details, :code:`CountWingAtoms`is declared as a submodel that fulfills the role of "wing atom
```

Comment:
Why deleting the space after :code:`CountWingAtoms` here and not after :code:`CountAtoms` below for instance ?

Reply:
Fixed in ae8a74c5196861503dc1ea7971f1a0e4da41eb08